src/app/error-handling/components/fatal-error/fatal-error.component.scss

```diff
@@ -0,0 +1,8 @@
+.fatal-error-page {
+  background-color: red;
+  color: white;
+
+  fatal-error-page__error-message {
```

Comment:
This should be a class.

Reply:
Eagle eye strikes again. Thanks! I'll fix it.